src/main/scala/com/danielasfregola/twitter4s/http/clients/streaming/statuses/TwitterStatusClient.scala

```diff
@@ -48,13 +49,13 @@ trait TwitterStatusClient {
     */
   def filterStatuses(follow: Seq[Long] = Seq.empty,
                      tracks: Seq[String] = Seq.empty,
-                     locations: Seq[Double] = Seq.empty,
+                     locations: GeoBoundingBoxes = GeoBoundingBoxes.empty,
```

Comment:
Why not simply have `Seq[GeoBoundingBox]` defaulted to Seq.empty? I think we do not actually need `GeoBoundingBoxes.empty` here...